Header header
geometry_msgs/Twist body_velocity
float64[] body_inertia_matrix
float64 body_mass
duration time_delta
geometry_msgs/Pose body_com
geometry_msgs/Point ground_contact
geometry_msgs/Point contact_position
geometry_msgs/Vector3 contact_normal
humanoid_catching/Limit[] torque_limits
float64[] jacobian_matrix

---
Header header
float64[] torques

